sites/partners/src/components/listings/PaperListingForm/sections/Units.tsx

```diff
@@ -129,43 +193,119 @@ const FormUnits = ({ units, setUnits, disableUnitsAccordion, featureFlags }: Uni
     }
   }
 
+  function saveUnitGroup(newUnitGroup: TempUnitGroup) {
+    const exists = unitGroups.some((unitGroup) => unitGroup.tempId === newUnitGroup.tempId)
```

Comment:
issue: whenever I try to save a second unit it always replaces the first unit. The issue seems to be "tempId" for both unit groups (the value is `1`)